zetaclient/tss/healthcheck.go

```diff
@@ -24,13 +27,17 @@ type HealthcheckProps struct {
 	Interval                time.Duration
 	WhitelistPeers          []peer.ID
 	NumConnectedPeersMetric prometheus.Gauge
+	NumKnownPeersMetric     prometheus.Gauge
 }
 
 // HealthcheckWorker checks the health of the TSS server and its peers.
 func HealthcheckWorker(ctx context.Context, server *tss.TssServer, p HealthcheckProps, logger zerolog.Logger) error {
 	if p.NumConnectedPeersMetric == nil {
 		return errors.New("missing NumConnectedPeersMetric")
 	}
+	if p.NumKnownPeersMetric == nil {
```

Comment:
You need to add this metric `healthCheckProps` in `setup.go`

Reply:
I removed this as it's not particularly useful.